### ISSUE TITLE
Lower EV Smart Charging Minimum EV SOC does not change duration of expected charging

### ISSUE THREAD

**Issue by Ferrox85** [jonasbkarlsson/ev_smart_charging#264]
Version 1.10.0

If for example 50 % of charging is set as goal. I expect that the duration is lowered with 50%. For example from 10 (normally 100%) to 5 hours and that it recalculates the most efficient hours to charge with those 5 hours.

**Comment by jonasbkarlsson** [comment]
Well, it depends on the current SOC. If your current SOC is 0%, then your example is what will happens. But if you for example have 25% SOC and it would take 10 hours to reach 100% (from 25%), then if you change the target SOC to 50%, then it will take (50-25)/(100-25)*10 = 3.33 hours (from 25% to 50%), which the integration rounds up to 4 hours.

**Comment by Ferrox85** [comment]
Exactly, but it doesn't do recalculation that when I change the target. So it looks like a bug right?

**Comment by yoali11** [comment]
It is the same for me. Whenever the target soc slider is changed nothing is recalculated. I have to manually turn off and back on the smart charge enable switch to have the integration recalculate a new charge plan.
Actually I have an automation that does this for me. 
Would be nice to have this embedded natively in the integration.

**Comment by jonasbkarlsson** [comment]
Ah, ok. I will check.

**Comment by jonasbkarlsson** [comment]
In [v1.11.0](https://github.com/jonasbkarlsson/ev_smart_charging/releases/tag/v1.11.0), changing the target SOC will update the charging plan.

Was that the issue, or was there also something else? Hmm. Is it the same problem when changing `Minimum EV SOC`?

**Comment by Ferrox85** [comment]
I don't use the minimum setting, but maybe? I can't test it right now..

**Comment by Ferrox85** [comment]
Just updated to this version, it does not fix the issue. I change the entity "EV Smart Charging Minimum EV SOC" Sorry for the inconvenience, I thought it was an other entity. This is because of some coupled automations and helpers in my HASS.

If I change continues charging for example, it immediately recalculates. But changing the EV Smart Charging Minimum EV SOC entity does not update.

**Comment by Ferrox85** [comment]
If I am able to try again I will report back. Currently our car is full...
